aws-test/tests/aws_audit_manager_framework/test-turbot-expected.json

```diff
@@ -0,0 +1,6 @@
+[
+	{
+		"akas": ["{{ output.arn.value }}"],
+		"title": "{{ output.name.value }}"
+	}
+]
```

Comment:
@bigdatasourav Seems like the file indented with 4 spaces. Can you please check your vscode settings?